packages/ses/test/error/tame-v8-error.test.js

```diff
@@ -1,7 +1,7 @@
 import test from 'ava';
 import '../../index.js';
 
-lockdown();
+lockdown({ errorTaming: 'safe' });
```

Comment:
The absence of this was a previously unnoticed bug, since omitting it left it up to environment variables to determine, which ruins the purpose of the test. Since the relevant env variables would not be set in CI, the bug would not be noticed there either.

Reply:
Resolving.